financial-connections/src/main/java/com/stripe/android/financialconnections/features/success/SuccessContent.kt

```diff
@@ -84,14 +87,20 @@ internal fun SuccessContent(
 private fun SuccessContentInternal(
     overrideAnimationForPreview: Boolean,
     payloadAsync: Async<Payload>,
+    topAppBarState: TopAppBarState,
     onCloseClick: () -> Unit,
     completeSessionAsync: Async<FinancialConnectionsSession>,
     onDoneClick: () -> Unit
 ) {
+    val topAppBarHost = LocalTopAppBarHost.current
     val scrollState = rememberScrollState()
     var showSpinner by remember { mutableStateOf(overrideAnimationForPreview.not()) }
     val payload by remember(payloadAsync) { mutableStateOf(payloadAsync()) }
 
+    LaunchedEffect(scrollState.canScrollBackward) {
+        topAppBarHost.updateTopAppBarElevation(isElevated = scrollState.canScrollBackward)
+    }
```

Comment:
might be missing something here: in which situation should the top app bar be elevated on success pane?

Reply:
Probably only in landscape mode on a small screen, when the content can’t fit? If we don’t think that’s a big concern, I can fully remove it in this screen.

Reply:
yeah I think we can remove it for simplicity : )